.github/workflows/docker.yml

```diff
@@ -171,11 +171,6 @@ jobs:
     runs-on: ubuntu-20.04
     needs:
       - build
-    strategy:
-      fail-fast: false
-      matrix:
-        # run copies of the current job in parallel
-        containers: [1, 2, 3, 4]
```

Comment:
Why are you getting rid of the parallel testing? Seems to speed things up quite a bit no?

Reply:
It seems it increases the flakiness of the test, probably due to the start-up time of the frontend. Even looking at the cypress dashboard didn't help, the recording starts at random steps. I leave this for a different pr.